src/main/kotlin/no/nav/familie/ef/sak/behandling/NyeBarnService.kt

```diff
@@ -11,34 +13,61 @@ import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.gjeldende
 import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.identer
 import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.visningsnavn
 import no.nav.familie.kontrakter.felles.PersonIdent
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.dao.DuplicateKeyException
+import org.springframework.data.relational.core.conversion.DbActionExecutionException
 import org.springframework.stereotype.Service
 import java.util.UUID
 
 @Service
 class NyeBarnService(private val behandlingService: BehandlingService,
                      private val fagsakService: FagsakService,
                      private val personService: PersonService,
-                     private val barnService: BarnService) {
+                     private val barnService: BarnService,
+                     private val taskRepository: TaskRepository) {
+
+    private val logger: Logger = LoggerFactory.getLogger(javaClass)
 
     fun finnNyeBarnSidenGjeldendeBehandlingForPersonIdent(personIdent: PersonIdent): List<String> {
         val personIdenter = personService.hentPersonIdenter(personIdent.ident).identer()
         val fagsak = fagsakService.finnFagsak(personIdenter, Stønadstype.OVERGANGSSTØNAD)
                      ?: error("Kunne ikke finne fagsak for personident")
 
-        return finnNyeBarnSidenGjeldendeBehandlingForFagsak(fagsak.id).map { it.personIdent }
+        val finnNyeBarnSidenGjeldendeBehandlingForFagsak = finnNyeBarnSidenGjeldendeBehandlingForFagsak(fagsak.id)
+
+        opprettOppfølgningsoppgaveForBarn(fagsak, finnNyeBarnSidenGjeldendeBehandlingForFagsak)
+
+        return finnNyeBarnSidenGjeldendeBehandlingForFagsak.map { it.personIdent }
+    }
+
+    // TODO slett 4 måneder etter att siste migreringen er klar
+    private fun opprettOppfølgningsoppgaveForBarn(fagsak: Fagsak, nyeBarn: List<BarnMinimumDto>) {
+        if (fagsak.migrert) {
```

Comment:
Håndterer vi de ikke-migrerte på en annen måte?
Går litt i surr for meg hva som skjer når :)

Reply:
Ikke-migrerte skal bli revurdert hvis det er ett nytt barn, og då får vi inn det nye barnet